basic-javascript/project-2.js

```diff
@@ -10,10 +10,11 @@ const getBiggest = (x, y) => {
 };
 
 const greeting = language => {
-  switch (language) {
-    case 'German':
+  let lang = language.toLowerCase();
+  switch (lang) {
+    case 'german':
       return 'Guten Tag!';
-    case 'Spanish':
+    case 'spanish':
       return 'Hola!';
     default:
       return 'Hello!';
```

Comment:
Nice job converting everything to lowercase so that the user can enter either without it messing up the test results.

Reply:
Thanks, Kelly.